.gitmodules

```diff
@@ -1,3 +1,7 @@
 [submodule "third_party/benchmark"]
 	path = third_party/benchmark
 	url = https://github.com/google/benchmark.git
+[submodule "third_party/googletest"]
```

Comment:
Can you pin this submodule to a specific googletest version (a tag or a commit)?

Reply:
I believe it is already.  The SHA1 of the commit a submodule points to is stored directly in the git database:

https://stackoverflow.com/a/5033973

You can see that it already points at a SHA1 here:

https://github.com/coryan/protobuf/tree/fix-issue-236-gmock-as-submodule/third_party